lang_java/parsing/ast_java.ml

```diff
@@ -289,6 +289,8 @@ and stmt =
 
   (* decl as statement *)
   | LocalVar of var_with_init
```

Comment:
Why not just change it to LocalVars of var_with_init list?
and remove LocalVar?

Reply:
This makes it easier to do the flattening later